hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemChooseSAS.java

```diff
@@ -128,11 +130,18 @@ public void testOnlyFixedTokenConfigured() throws Exception {
     try (AzureBlobFileSystem newTestFs = (AzureBlobFileSystem)
         FileSystem.newInstance(testAbfsConfig.getRawConfiguration())) {
 
-      // Asserting that account SAS is used as both filesystem and blob level operations succeed.
-      newTestFs.getFileStatus(new Path("/"));
-      Path testPath = new Path("/testCorrectSASToken");
-      newTestFs.create(testPath).close();
-      newTestFs.delete(new Path("/"), true);
+      // Asserting that FixedSASTokenProvider is used.
+      Assertions.assertThat(testAbfsConfig.getSASTokenProvider())
+          .describedAs("Custom SASTokenProvider Class must be used")
+          .isInstanceOf(FixedSASTokenProvider.class);
+
+      // Assert that Account SAS is used and only read operations are permitted.
```

Comment:
Why was create passing in the last test case and would give Access Denied exception now ?